modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -25,6 +26,13 @@ Modify the following vSphere configuration settings as required:
 . In the Administrator perspective, navigate to *Home -> Overview*.
 . Under *Status*, click *vSphere connection* to open the *vSphere connection configuration* wizard.
 . In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *vCenter cluster* field, enter the name of the vSphere vCenter cluster where {product-title} is installed.
++
+[NOTE]
```

Comment:
Per the supplementary style guide's section on admonitions, this reads more like an [IMPORTANT](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions:~:text=performance%2C%20or%20supportability.-,IMPORTANT,-Advisory%20information%20essential) admonition to me.

However, it's tricky because I suppose this note might not be "essential" to users with 4.12 clusters.

Reply:
I'll change it to Important. They can't really make a mistake though, as the asterisk will be there from 4.13 onwards. It's more to tell users of earlier versions that they can skip that field if they don't have the cluster name.

Reply:
I've also added 4.14 to the merge versions. Thanks for your feedback!